packages/listview/src/components/FloatingSearch/__test__/floatingSearch.spec.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import { mount } from 'enzyme';
+import Search from '../search';
+
+describe('<FloatingSearch />', () => {
+    it('should render an input type search and a button', () => {
+        const component = mount(<Search />);
+        const input = component.find('input');
+        const button = component.find('button');
+        expect(input.prop('type')).toBe('search');
+        expect(button.exists()).toBe(true);
+    });
+
+    it('should render an input with value = a and 2 buttons', () => {
+        const component = mount(<Search value="a" />);
+        const input = component.find('input');
+        const buttons = component.find('button');
+        expect(input.prop('value')).toBe('a');
+        expect(buttons.length).toBe(2);
+    });
+
+    it('should call onChage with value = b', () => {
+        const onChangeMockFn = jest.fn();
+        const component = mount(<Search value="a" onChange={onChangeMockFn} />);
+        const input = component.find('input');
+        input.simulate('change', { target: { value: 'b' } });
+        expect(onChangeMockFn.mock.calls.length).toBe(1);
+        expect(onChangeMockFn).toBeCalledWith('b');
```

Comment:
here is sufficient with this last expect since if it was called with 'b' then it was called